mne/viz/_3d.py

```diff
@@ -1552,6 +1571,21 @@ def _plot_mpl_stc(stc, subject=None, surface='inflated', hemi='lh',
         time_viewer.subplots_adjust(left=0.12, bottom=0.05, right=0.75,
                                     top=0.95)
     fig.subplots_adjust(left=0., bottom=0., right=1., top=1.)
+
+    # add colorbar
+    from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+    sm = plt.cm.ScalarMappable(cmap=cmap,
+                               norm=plt.Normalize(0, ctrl_pts[2]))
```

Comment:
okay I _think_ I got it to work. But there is a hack here that I don't understand at the moment. There is probably a better way to do this ...